src/plugins/ai_assistant_management/selection/public/routes/components/ai_assistant_selection_page.tsx

```diff
@@ -90,7 +90,7 @@ export function AiAssistantSelectionPage() {
                 <p>
                   {i18n.translate(
                     'aiAssistantManagementSelection.aiAssistantSelectionPage.obsAssistant.documentationLinkDescription',
-                    { defaultMessage: 'For more info, see our' }
+                    { defaultMessage: 'For more info, refer to our' }
```

Comment:
I assume somebody else wrote this code, but we should use FormattedMessage here to render links as part of the message (e.g. what if in another language documentation comes earlier in the sentence?), and the doc links service (CoreStart.docLinks) - this always links to the current/latest version of the documentation, but the current deployment might be an older version.